mautrix_signal/db/puppet.py

```diff
@@ -203,6 +204,13 @@ async def all_with_custom_mxid(cls) -> list[Puppet]:
         return [cls._from_row(row) for row in rows]
 
     @classmethod
-    async def all_with_initial_activity(cls) -> list["Puppet"]:
-        rows = await cls.db.fetch(f"{cls._select_base} WHERE first_activity_ts is not NULL")
-        return [cls._from_row(row) for row in rows]
+    async def recently_active_count(cls, min_activity_days: int, max_activity_days: int | None) -> int:
+        current_ms = time.time() * 1000
+        query = (
+            "SELECT COUNT(*) FROM puppet WHERE (last_activity_ts - first_activity_ts) > $2"
```

Comment:
The standard query only contains `$2` but the addition has `$1` and `$3`?
The ordering seems off.

Reply:
Yeah, it was a little stupid like that – I got it straight from mautrix-telegram. I reordered it now to make it a bit saner.